src/coroutine.js

```diff
@@ -38,7 +38,9 @@ function processGenerator (action, body, repo) {
  * Coroutine is used by an action to determine how it should resolve
  * the body of their associated command.
  */
-export default function coroutine (action, body, repo) {
+export default function coroutine (action, command, params, repo) {
+  let body = command.apply(action, params)
```

Comment:
noticed this went from `command.apply(null, params)` to `command.apply(action, params)`, why is that?